lib/livebook/hubs/team_client.ex

```diff
@@ -166,6 +169,7 @@ defmodule Livebook.Hubs.TeamClient do
 
   defp handle_event(:user_connected, user_connected, %{secrets: []} = state) do
     secrets = for secret <- user_connected.secrets, do: build_secret(state, secret)
+    for secret <- secrets, do: Broadcasts.secret_created(secret)
 
     %{state | secrets: secrets}
   end
```

Comment:
Let's remove this clause. I think the other branch will be quick enough anyway? And less chance of having errors like these. :)

```suggestion
```